smac/intensification/simple_intensifier.py

```diff
@@ -30,36 +31,12 @@ class RandomSearcher(AbstractRacer):
         cutoff of TA runs
     deterministic : bool
         whether the TA is deterministic or not
-    initial_budget : typing.Optional[float]
-        minimum budget allowed for 1 run of successive halving
-    max_budget : typing.Optional[float]
-        maximum budget allowed for 1 run of successive halving
-    eta : float
-        'halving' factor after each iteration in a successive halving run. Defaults to 3
-    num_initial_challengers : typing.Optional[int]
-        number of challengers to consider for the initial budget. If None, calculated internally
     run_obj_time : bool
         whether the run objective is runtime or not (if true, apply adaptive capping)
-    n_seeds : typing.Optional[int]
-        Number of seeds to use, if TA is not deterministic. Defaults to None, i.e., seed is set as 0
-    instance_order : typing.Optional[str]
-        how to order instances. Can be set to: [None, shuffle_once, shuffle]
-        * None - use as is given by the user
-        * shuffle_once - shuffle once and use across all SH run (default)
-        * shuffle - shuffle before every SH run
-    adaptive_capping_slackfactor : float
-        slack factor of adpative capping (factor * adaptive cutoff)
-    inst_seed_pairs : typing.List[typing.Tuple[str, int]], optional
-        Do not set this argument, it will only be used by hyperband!
+    adaptive_capping_slackfactor: float
```

Comment:
As there won't be any adaptive capping in this intensifier, could you drop this argument and just assign a value of 1?

Reply:
When creating the intensifier, the current api provide this parameter, because the base class requires it. If you remove it you get:
```
TypeError: __init__() got an unexpected keyword argument 'cutoff'

```